packages/@apphosting/adapter-angular/package.json

```diff
@@ -22,7 +22,9 @@
     "sideEffects": false,
     "scripts": {
         "build": "rm -rf dist && tsc && chmod +x ./dist/bin/*",
-        "test": "ts-mocha -p tsconfig.json src/**/*.spec.ts"
+        "test": "ts-mocha -p tsconfig.json src/**/*.spec.ts",
+        "registry:local": "npx verdaccio --config ../publish-dev/verdaccio-config.yaml",
+        "publish:local": "npm unpublish --@apphosting:registry=http://localhost:4873 --force && npm publish --@apphosting:registry=http://localhost:4873"
```

Comment:
The unpublish causes some issues on the first run since there is nothing to unpublish, I get this error Not Found - GET http://localhost:4873/@apphosting%2fadapter-nextjs?write=true - no such package available

Reply:
also running into this issue http://localhost:4873/@apphosting%2fadapter-angular?write=true